sales_billing/hooks.py

```diff
@@ -90,12 +68,12 @@
 # ------------
 
 # before_install = "sales_billing.install.before_install"
-# after_install = "sales_billing.install.after_install"
+after_install = "sales_billing.install.after_install"
 
 # Uninstallation
 # ------------
 
-# before_uninstall = "sales_billing.uninstall.before_uninstall"
+before_uninstall = "thai_payroll.setup.before_uninstall"
```

Comment:
Why are you referencing thai_payroll?
```suggestion
before_uninstall = "sales_billing.setup.before_uninstall"
```